vignettes/kegg.Rmd

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
I started commenting on the wrong file before we talked through the precompiling thing, sorry!